lib/collection/src/shards/local_shard/clock_map.rs

```diff
@@ -39,6 +39,13 @@ impl ClockMap {
         Ok(())
     }
 
+    #[cfg(test)]
+    pub fn get_tick(&self, peer_id: PeerId, clock_id: u32) -> Option<u64> {
+        self.clocks
+            .get(&Key::new(peer_id, clock_id))
+            .map(|clock| clock.current_tick)
+    }
+
```

Comment:
There's already `#[cfg(test)]` only `current_tick` after you rebased on `dev`. %)

Reply:
Good catch. Thanks!